docs/rfcs/1007-frozen-users.md

```diff
@@ -0,0 +1,133 @@
+
+## HTTP `users` route
+
+This route is made available as `/administration/organizations/<raw_organization_id>/users` and requires an administration token.
+
+### `GET` method
+
+The `GET` method is used to list information for all users, including parsec ID, user name, user email and frozen status.
+
+The body of the `GET` response is a JSON structure with the following format:
+
+```jsonc
+[
+    {
+        "user_id": "<parsec_user_id_1>",
+        "user_name": "<user_name_1>",
+        "user_email": "<user_email_1>",
+        "frozen": true // or false
+    },
+    // [...]
+    {
+        "user_id": "<parsec_user_id_N>",
+        "user_name": "<user_name_N>",
+        "user_email": "<user_email_N>",
+        "frozen": true // or false
+    }
+]
+```
+
+The list contains all users of the organization, including those that are not frozen.
+
+However, it does not contain the users that have been revoked.
+
+## HTTP `users/frozen` route
+
+This route is made available as `/administration/organizations/<raw_organization_id>/users/frozen` and requires an administration token.
+
+### `PATCH` method
+
+The `PATCH` method is used to modify the `frozen` status for a given user.
+
+The body of the `PATCH` request must be a JSON structure with either one of following formats:
+
+```jsonc
+{
+    "user_email": "<user_email>",
+    "frozen": true // or false
+}
+```
+
+Or:
+
+```jsonc
+{
+    "user_id": "<parsec_user_id>",
+    "frozen": true // or false
+}
+```
+
+This way, a user can either be identified by its email or by its parsec ID.
+
+The body of the `PATCH` response is a JSON structure summarizing the updated information for the selected user:
+
+```jsonc
+{
+    "user_id": "<parsec_user_id>",
+    "user_name": "<user_name>",
+    "user_email": "<user_email>",
+    "frozen": true // or false
+}
+```
+
+### User identification: email address vs parsec ID
+
+There is a subtle difference between the two ways to identify a user. At any given time, an email address can be used to uniquely identify an non-revoked user from a given organization. In contrast, a parsec user ID identifies uniquely any user from all organizations in the parsec server, including revoked users. This means that over time, an email address can identify different parsec users with different parsec IDs, even from the same organization.
+
+The frozen status configured by the `PATCH` method is specifically associated with the parsec user ID, regardless of the identification method used in the request body. This has a the following consequence: if a user is revoked and then a new user is created with the same email address, the frozen status will **not** be applied to the new user.
```

Comment:
I understand the differences between email address and parsec ID, but I am not sure I follow the logic here... 

Regarding the _identification_ of the user to be frozen : 
The requirement is to _freeze a user from a specific organization_. The organization is already identified from the endpoint (`/administration/organizations/<raw_organization_id>/users/frozen`) therefore the email address seem sufficient to uniquely _identify_ the user to freeze (and the identification it via parsec ID in the PATCH request seem redundant).  

Regarding the "attaching" of the frozen status to the email vs parsec ID:
I am not convinced that we want to allow the new user (with same email) to `not` be frozen... and in any case, this could be implemented during revocation (i.e. clearing frozen status for the corresponding email address).